src/index.ts

```diff
@@ -1,11 +1,12 @@
 import { Observable } from "rxjs/Observable";
+import { from } from "rxjs/observable/from";
 import { IpcMark } from "./common/types";
 
-import mainProcessMonitor from "./main";
-import rendererProcessMonitor from "./renderer";
-
-const ipcMonitor: Observable<IpcMark> =
-  process?.type === "renderer" ? rendererProcessMonitor : mainProcessMonitor;
+const ipcMonitor: Observable<IpcMark> = from(
+  process?.type === "renderer"
+    ? import("./renderer").then((module) => module.default)
+    : import("./main").then((module) => module.default)
+).mergeAll();
```

Comment:
this is a quirk of RxJS v5, but you'll also need to add an `import "rxjs/add/operator/mergeAll";` to the file :{

Reply:
its their way of helping tree-shake